src/website/views/components/activity.pug

```diff
@@ -0,0 +1,1105 @@
+              <div>04/22/2019</div>
+            </div>
+            <div class="m-activity__item-avatar">
+              <div class="a-avatar">
+                <img src="../../assets/images/avatar.jpg" alt="avatar">
+              </div>
+            </div>
+            <div class="m-activity__end">
+              <div class="m-activity__end-header">Password change for user trent.anderson</div>
+              <div class="m-activity__end-footer">CTlops-Admin</div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="m-activity__day">
+        <div class="m-activity__header">Monday, Apr 21 2019</div>
+        <div class="m-activity__content">
+          <div class="m-activity__item">
+            <div class="m-activity__start">
+              <div>04/22/2019</div>
+            </div>
+            <div class="m-activity__item-avatar">
+              <div class="a-avatar">
+                <img src="../../assets/images/avatar.jpg" alt="avatar">
+              </div>
+            </div>
+            <div class="m-activity__end">
+              <div class="m-activity__end-header">Password change for user trent.anderson</div>
+              <div class="m-activity__end-footer">CTlops-Admin</div>
+            </div>
+          </div>
+          <div class="m-activity__item">
+            <div class="m-activity__start">
+              <div>04/22/2019</div>
+            </div>
+            <div class="m-activity__item-avatar">
+              <div class="a-avatar">
+                <img src="../../assets/images/avatar.jpg" alt="avatar">
+              </div>
+            </div>
+            <div class="m-activity__end">
+              <div class="m-activity__end-header">Password change for user trent.anderson</div>
+              <div class="m-activity__end-footer">CTlops-Admin</div>
+            </div>
+          </div>
+          <div class="m-activity__item">
+            <div class="m-activity__start">
+              <div>04/22/2019</div>
+            </div>
+            <div class="m-activity__item-avatar">
+              <div class="a-avatar">
+                <img src="../../assets/images/avatar.jpg" alt="avatar">
+              </div>
+            </div>
+            <div class="m-activity__end">
+              <div class="m-activity__end-header">Password change for user trent.anderson</div>
+              <div class="m-activity__end-footer">CTlops-Admin</div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+h4 Feed
+p.-text Apply the modifier class <code>-feed</code> to render with feed style
+.example.-mb--3
+  .-p--5
+    .m-activity.-feed
+      .m-activity__day
+        .m-activity__content
+          .m-activity__item
+            .m-activity__item-avatar
+              .a-avatar
+                img(src='../../assets/images/avatar.jpg', alt="avatar")
+            .m-activity__end
+              .m-activity__end-header
+                strong trent.anderson
```

Comment:
The name (e.g. trent.anderson) and the artifact (e.g. Active Directory) could be plain text or a link. We need to ensure that whether it's a `strong` or an `a` tag, it renders with a `font-weight: 600;` (semi-bold).